addons/binding/org.openhab.binding.tankerkoenig/META-INF/MANIFEST.MF

```diff
@@ -19,6 +19,7 @@ Import-Package:
  org.joda.time.base,
  org.openhab.binding.tankerkoenig,
  org.openhab.binding.tankerkoenig.handler,
+ org.osgi.framework;version="1.8.0",
```

Comment:
Do you need this specific version of the OSGI framework? If not, please remove the version constraint.